specifications/api/swagger.yaml

```diff
@@ -236,7 +236,7 @@ x-syncClockProgress: &syncClockProgress
     status: pending
 
 x-amount: &amount
-  description: Coins, in Lovelace
+  description: Coins, in Lovelace. Only relates to 'Ada'. Refer to `assets` for multi-assets wallets instead.
```

Comment:
Q: Can a user specify to send 0 ada, and 1 Apple asset?

I'm guessing: yes, but the coin selection will automatically send the minimum required ada along in the token bundle?

Reply:
Indeed. And also assign the minimum ada value needed for the multi asset UTxO.